### PR TITLE
Documented the background process

### LINKED ISSUE #1567: Document background process
We have removed crono in favor of a background process that runs the same job (among other things). We should document this.

### PR DESCRIPTION
**Pull request by mssola**
Fixes #1567

Signed-off-by: Miquel Sabaté Solà <msabate@suse.com>

### CODE REVIEW

File: _docs/background.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  Only available in <strong>2.3 or later</strong>.
+</div>
+
+# What's the background process?
+
+Every production-ready deployment of Portus must have a process/container called
+background which performs some needed tasks for the normal operation of Portus.
+
+If you are running bare metal, you can simply run this process from the source
+code like this:
```
Comment (vitoravelino) [inline _docs/background.md]: Maybe ommit `this` or replace it with `below`?

File: _docs/background.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+```
+$ bundle exec rails r bin/background.rb
+```
+
+That being said, unless you are in a development environment, you won't have to
+perform this command. Instead, if you are using the official [Docker
```
Comment (vitoravelino) [inline _docs/background.md]: `that`?

File: _docs/background.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+image](https://hub.docker.com/r/opensuse/portus/tags/), you will have to set the
+following environment variable like this:
```
Comment (vitoravelino) [inline _docs/background.md]: Omit?

File: _docs/background.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+`PORTUS_INIT_COMMAND=bin/background`. This has been already set in the [examples
+that we provide](https://github.com/SUSE/Portus/tree/master/examples).
+
+## Tasks
+
+As documented above, this background process consists of some tasks that have to
```
Comment (vitoravelino) [inline _docs/background.md]: `the` instead of `this`? Trying to avoid the amount of `this`.

File: _docs/background.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+blocking issues.
+
+### Registry synchronization
+
+All Docker registries provide an API in which any client can fetch some
+information. Portus makes a heavy use of this API, and in this case it fetches
+the catalog of Docker images/tags. This is done periodically, and it will update
+the database when needed.
+
+This way, Portus makes sure that there are no consistency issues between the
```
Comment (vitoravelino) [inline _docs/background.md]: Remove comma?

**Comment by mssola** [comment]
@vitoravelino fixed

**Pull request merged**